src/main/java/Duke.java

```diff
@@ -6,5 +11,44 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        Scanner in = new Scanner(System.in);
+        String input;
+        ArrayList<String> items = new ArrayList<String>();
+        ArrayList<Boolean> marked = new ArrayList<Boolean>();
+        
+        while((input = in.nextLine()) != "") {
+            if (input.equalsIgnoreCase("bye")) {
```

Comment:
I recommend standardising between input.equals() and input.equalsIgnoreCase() for bye and other commands, so as to make the code easier to understand and debug incase an error arises during testing.